common/changes/@typespec/compiler/feature-lsp-perf_2023-12-19-21-45.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "IDE: Performance improvements to the language server. TypeSpec extension will not recompile on every char added but will resolve diagnostics only after a debounce timer.",
```

```suggestion
      "comment": "IDE: Performance improvements to the language server."
```

Personally, I think the implementation details aren't so important for the changelog.

---

yeah that's fair, I feel like though I see to commonly changelog entries where it just says "better perf" and you don't see where